### PR TITLE
Made some variable names more consistent with the other parts.

### PR DESCRIPTION
**Pull request by euske**
Hello, we're developing an automated system that detects inconsistent variable names in a large software project. Our system checks if each variable name is consistent with other variables in the project in its usage pattern, and proposes correct candidates if inconsistency is detected. This is a part of academic research that we hope to publish soon, but as a part of the evaluation, we applied our systems to your projects and got a few interesting results. We carefully reviewed our system output and manually created a patch to correct a few variable names. We would be delighted if this patch is found to be useful. If you have a question or suggestion regarding this patch, we'd happily
answer. Thank you.

P.S. our patch is purely for readability purposes and does not change any functionality. A couple of issues that we've noticed were left untouched. For example, the use of variable names "len" and "length" were fairly widespread, but we have only corrected a few notable instances to minimize our impact.

### CODE REVIEW

File: src/main/java/org/apache/bcel/classfile/LocalVariableTypeTable.java [hunk 1]
```diff
@@ -67,8 +67,8 @@ public LocalVariableTypeTable(final int name_index, final int length, final Loca
         this.local_variable_type_table = local_variable_table;
     }
 
-    LocalVariableTypeTable(final int nameIdx, final int len, final DataInput input, final ConstantPool cpool) throws IOException {
-        this(nameIdx, len, (LocalVariable[]) null, cpool);
+    LocalVariableTypeTable(final int name_index, final int length, final DataInput input, final ConstantPool cpool) throws IOException {
+        this(name_index, length, (LocalVariable[]) null, cpool);
```
Comment (garydgregory) [inline src/main/java/org/apache/bcel/classfile/LocalVariableTypeTable.java]: I am -1 to using underscores in Java. If anything,  we should change existing underscores to camel case. I like changing idx to index.

File: src/main/java/org/apache/bcel/verifier/structurals/Subroutines.java [hunk 2]
```diff
@@ -121,16 +121,16 @@ public String toString() {
             ret.append("', Instructions: '").append(instructions).append("'.");
 
             ret.append(" Accessed local variable slots: '");
-            int[] alv = getAccessedLocalsIndices();
-            for (final int element : alv) {
-                ret.append(element);ret.append(" ");
+            int[] lvs = getAccessedLocalsIndices();
+            for (final int lv : lvs) {
```
Comment (garydgregory) [inline src/main/java/org/apache/bcel/verifier/structurals/Subroutines.java]: I do not see how trading an acronym for a mysterious TLA helps.

File: src/main/java/org/apache/bcel/classfile/LocalVariableTypeTable.java (hunk 1 above)
Comment (euske) [reply src/main/java/org/apache/bcel/classfile/LocalVariableTypeTable.java]: Although I'm not insisting, *_index is a pretty common naming scheme throughout the project (about 600 instances). Actually *idx is common too (about 200 instances), but the other constructor in this class already uses name_index as an argument, so I thought this change fits more naturally than the other way around.

File: src/main/java/org/apache/bcel/verifier/structurals/Subroutines.java (hunk 2 above)
Comment (garydgregory) [reply src/main/java/org/apache/bcel/verifier/structurals/Subroutines.java]: To make things worse :-) this also trades a word "element" for a meaningless acronym "lv" in this context.

Comment (euske) [reply src/main/java/org/apache/bcel/verifier/structurals/Subroutines.java]: The rationale is that "lvs" (apparently LocalVariables) is consistently used in several modules in verifier/structurals. "alv" was an outlier here.

File: src/main/java/org/apache/bcel/classfile/LocalVariableTypeTable.java (hunk 1 above)
Comment (garydgregory) [reply src/main/java/org/apache/bcel/classfile/LocalVariableTypeTable.java]: Unless you plan on following up with changing `nameIdx` to `nameIndex`, we should close this PR. Please advise.

**Comment by garydgregory** [comment]
No action, closing.

**Pull request closed without merging**